cartography/data/jobs/analysis/aws_account_to_organization_link.json

```diff
@@ -0,0 +1,13 @@
+{
+  "statements": [
```

Comment:
Rather than do this as part of an analysis job, could you move it to the Python code? My thinking is that analysis jobs are sort of "extras" with cartography and things like the relationship between an organization and account seems foundational, so we should have it in code because not all users of cartography will opt to run analysis jobs.

I'm open to thoughts here though.

Reply:
@achantavy sorry we haven't been following this up.  We'll address the issues by the end of the week.